templates/JonoM/BetterNavigator/BetterNavigator.ss

```diff
@@ -43,98 +44,96 @@
 
         <% if $Mode == 'dev' || $IsDeveloper %>
 
-            <div class="bn-heading"><%t JonoM\BetterNavigator.DEVELOPER_TOOLS_HEADING 'Developer tools' %></div>
+        <h2 class="bn-heading"><%t JonoM\BetterNavigator.DEVELOPER_TOOLS_HEADING 'Developer tools' %></h2>
```

Comment:
I know it's not semantic, but divs were used here to avoid picking up any default styles from the project. If you want to use semantic elements you might need to apply a reset to each.

Reply:
ahh, ok, yes I hadn't thought of that.

I have added a few extra resets to various elements. Because everything is under an `id` selector, we have pretty high specificity so there actually weren't too many things to add.